### DISCUSSION TITLE
Can't make Public to Private Reposatory

### DISCUSSION THREAD

**Discussion by Bhoominv** [community/community#50741] [Repositories]
### Select Topic Area

General

### Body

Can you help me to change my repository public to private ?

I tried all things but not succeed. it shows me
Making this repository private could permanently erase these counts by removing stars and watchers associated to users that will no longer have access to this repository:
If you decide to make this repository public in the future, it will not be possible to restore these stars and watchers and this will affect its repository rankings.
Dependency graph and Dependabot alerts will remain enabled with permission to perform read-only analysis on this repository.
Code scanning will become unavailable.
Current forks will remain public and will be detached from this repository.

but there is no other option to make this private.

**Comment by davynpan** [comment]
Do you care about the stars and view counts?

**Comment by Bhoominv** [comment]
Hello,
Now it shows me a private. My problem solved. Sorry to not updated my
problem solve.
Thank you for your time.

**Comment by cehfisher** [comment]
Hi @Bhoominv - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category and close it since it has been resolved? Thanks!
